fusor-ember-cli/app/models/deployment.js

```diff
@@ -48,7 +48,7 @@ export default DS.Model.extend({
   updated_at: DS.attr('date'),
 
   // has one Engine
-  discovered_host: DS.belongsTo('discovered-host', {inverse: 'deployment', async: true}),
+  discovered_host: DS.belongsTo('discovered-host', {inverse: 'deployment'}),
 
```

Comment:
@knowncitizen, good call. I think that the default is becoming `true` is some future version so I'll make this `false`